### DISCUSSION TITLE
make up your mind

### DISCUSSION THREAD

**Discussion by afkvido** [Prodigy-Hacking/ProdigyMathGameHacking#1271] [(A) Anything]
## There's this in the [README](https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/blob/master/README.md):

![image](https://user-images.githubusercontent.com/69060894/149393520-5e23a195-5cb4-4713-af6d-d9c20b21620b.png)

## There's also this in #170 

![image](https://user-images.githubusercontent.com/69060894/149393682-f13b8c41-582e-43f1-99d9-9ed770bd5b43.png)

# make up your mind, people

**Comment by PatheticMustan** [comment]
how is this contradictory 

just don't post your account information online

**Reply by afkvido** [reply]
fair enough
